code/modules/customitems/item_defines.dm

```diff
@@ -426,6 +426,71 @@
 
 //////////// Masks ////////////
 
+obj/item/clothing/mask/gas/fluff/tyhailer // T0EPIC4U: Ty Omaha
+	name = "tacticool security gas mask"
+	desc = "A tacticool Security gas mask with modified 'Compli-o-nator 3000' device, it's labeled: Ty Omaha. It looks like it has some..residue..you probably don't want to touch it."
```

Comment:
I really don't want to do this, but you're forcing my hand here. 

`It's labeled: "Ty Omaha". `